specification/hanaonazure/resource-manager/readme.md

```diff
@@ -0,0 +1,68 @@
+# HanaOnAzure
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for HanaOnAzure.
+
+
+
+---
+## Getting Started 
+To build the SDK for HanaOnAzure, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+---
+
+## Configuration
+
+
+
+### Basic Information
+These are the global settings for the HANA on Azure API.
+
+``` yaml
+title: HanaManagementClient
+description: HANA on Azure Client
+openapi-type: arm
+tag: package-2017-11
+namespace: azure.mgmt.saphanaonazure
```

Comment:
`namespace`, `package-name` and `package-version` are programming language specific (C# could want the namespace "Azure.Management.SapHanaOnAzure"), so could you please move those 3 settings down into the `python` scope (e.g. after `payload-flattening-threshold`)? 🙂 

Just so we're not missing any potential quality issues: Did you see this in another RPs configuration?